app/assets/javascripts/application.js

```diff
@@ -23,4 +23,23 @@
 //= require_tree ./services
 //= require_tree ./controllers
 
-angular.module("OpenHq", ['ngRoute', 'ngAnimate', 'ngResource', 'ngSanitize']);
+angular.module("OpenHq", ['ngRoute', 'ngAnimate', 'ngResource', 'ngSanitize'])
+.config(function($routeProvider, $locationProvider) {
+  $routeProvider
+  .when('/', {
+    template: JST['templates/projects/index'],
```

Comment:
Using the JST style templates allows us to prepackage all the templates up in our JS and angular doesn’t need to ajax off to get the template when required like in gather